src/Horarium.InMemory/InMemoryRepositoryFactory.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using Horarium.Repository;
+
+namespace Horarium.InMemory
+{
+    public static class InMemoryRepositoryFactory
+    {
+        private static readonly Lazy<InMemoryRepository> 
+            Repository = new Lazy<InMemoryRepository>(() => new InMemoryRepository());
+
+        public static IJobRepository Create() => Repository.Value;
```

Comment:
The interface in the factory is not good because of the method `Create` every time return one repository.

Reply:
True this.
I guess then we should just remove the factory - `new InMemoryRepository` does the job. What say you?